debian/gen-flat-repo-fixtures.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+set -eux
+
+yes | apt-get install dpkg-dev apt-utils
```

Comment:
These build dependencies should go in the `debian-build` container in the `Containerfile`.
Ideally this script should not require root to run locally. :wink:

I will add a suggestion commit to show exactly what I mean.

Reply:
Thanks! I squashed your suggestion to the commit.